app/database.ts

```diff
@@ -65,6 +66,7 @@ export class Database_View implements ProfileSetter, ProfileGetter, EventRemover
     private readonly sourceOfChange = csp.chan<{ event: Parsed_Event; relay?: string }>(buffer_size);
     private readonly caster = csp.multi<{ event: Parsed_Event; relay?: string }>(this.sourceOfChange);
     private readonly profiles = new Map<string, Profile_Nostr_Event>();
+    private readonly deletionEvents = new Map</* event id */ string, /* deletion event */ Parsed_Event>();
```

Comment:
So this is a reverse index? The key of the map is the id of the deleted event?

Reply:
This part of the code has a high potential of refactoring in near future. Let's comment that this is a reverse index